### PR TITLE
Fix column name related issue for Forecast functions

### PR DESCRIPTION
**Pull request by xzdandy**
Addressing item3 in #1081

* [x] In `evadb/executor/create_function_executor.py`, we rename the input relationship to a [fixed schema](https://nixtla.github.io/statsforecast/docs/getting-started/getting_started_short.html) requested by statsforecast 
* [x] Rename the output column so it is synced with binder. A temporal fix. We will reconsider the design in #1017 
* [x] Update testcases to test the column rename feature.

### CODE REVIEW

**Comment by xzdandy** [comment]
@americast I am not able to request your review. Please take a look.

File: evadb/executor/create_function_executor.py [hunk 1]
```diff
@@ -219,24 +220,13 @@ def handle_forecasting_function(self):
...
-                function_name=None,
-                row_id=None,
-            ),
-            FunctionMetadataCatalogEntry(
-                key="model_path",
-                value=model_path,
-                function_id=None,
-                function_name=None,
-                row_id=None,
+                "output_column_rename", self.node.outputs[0].name
```
Comment (americast) [inline evadb/executor/create_function_executor.py:229]: May I know the difference between this and `"output_column_rename", arg_map["predict"]`? Thanks!

**Review by americast — APPROVED** [review]
Seems to work great @xzdandy. Thanks!

File: evadb/executor/create_function_executor.py (hunk 1 above)
Comment (xzdandy) [reply evadb/executor/create_function_executor.py:229]: That is a good point. We can use `arg_map["predict"]`.

File: evadb/executor/create_function_executor.py [hunk 2]
```diff
@@ -159,12 +159,13 @@ def handle_forecasting_function(self):
         model_name = arg_map["model"]
         frequency = arg_map["frequency"]
 
-        data = aggregated_batch.frames.rename(columns={arg_map["predict"]: "y"})
+        aggregated_batch.rename(columns={arg_map["predict"]: "y"})
```
Comment (jiashenC) [inline evadb/executor/create_function_executor.py:162]: Just for my knowledge, what is code 162-168 trying to achieve here?

Comment (xzdandy) [reply evadb/executor/create_function_executor.py:162]: The underlying statsforecast needs the column name to be the following:

1. The unique_id (string, int or category) represents an identifier for the series.
2. The ds (datestamp) column should be of a format expected by Pandas, ideally YYYY-MM-DD for a date or YYYY-MM-DD HH:MM:SS for a timestamp.
3. The y (numeric) represents the measurement we wish to forecast.

Comment (xzdandy) [reply evadb/executor/create_function_executor.py:162]: For reference: https://nixtla.github.io/statsforecast/docs/getting-started/getting_started_short.html

Comment (jiashenC) [reply evadb/executor/create_function_executor.py:162]: Sounds good. Can we add a reference in the code?

**Review by jiashenC — APPROVED** [review]

**Pull request merged**